nuget/System.Threading.Tasks.Channels.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>System.Threading.Tasks.Channels</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/dotnet/corefxlab</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Provides synchronization data structures for passing data between producers and consumers.</description>
+    <releaseNotes>Initial package</releaseNotes>
+    <copyright> Microsoft Corporation.  All rights reserved.</copyright>
+    <tags>.NET tasks channels corefxlab threading asynchrony async</tags>
+  </metadata>
+  <files>
+    <file src="..\bin\Windows_NT.AnyCPU.Release\System.Threading.Tasks.Channels\System.Threading.Tasks.Channels.dll" target="lib/portable-net45+win8+wpa81+aspnetcore50"/>
```

Comment:
Is there a specific reason you picked `portable-net45+win8+wpa81+aspnetcore50` as the target? As opposed to a generation? Also aspnetcore50 isn't even being used any longer.